extensions/wikia/ContentReview/services/ContentReviewService.class.php

```diff
@@ -10,7 +10,8 @@
 
 use Wikia\ContentReview\Models\ReviewModel,
 	Wikia\ContentReview\Models\ReviewLogModel,
-	Wikia\ContentReview\Models\CurrentRevisionModel;
+	Wikia\ContentReview\Models\CurrentRevisionModel,
+	Wikia\ContentReview\Helper;
```

Comment:
This shouldn't be needed as this service is also in the `Wikia\ContentReview` namespace.